docs/hazmat/primitives/mac/hmac.rst

```diff
@@ -28,10 +28,12 @@ of a message.
     .. doctest::
 
         >>> from cryptography.hazmat.primitives import hashes, hmac
+        >>> key = b'\xb4/\xc4\xbb/\xe3\xe8\x15\xc9\xd0\x80H \xe6\x10O\xf8\xcdI\xc4\xf8GN\xbd\xe9\x84\xa8e\xed\x0c&\xd0'
```

Comment:
I don't love this because based on past experience I suspect people will copy/paste this and start using the key. This is part of why we originally didn't directly show where the key was coming from. If we want to change that I'd suggest making a key that is very clearly non-random `key = b'test key. a real key should use os.urandom'` or something like that.

Reply:
OK, this way is somehow better than adding a comment that might be neglected by a copy-paste "programmer"  ;)